README.md

```diff
@@ -846,4 +846,67 @@ DELAY 500
 STRING :q!
 DELAY 500
 ENTER
-```
\ No newline at end of file
+```
+
+## The DevOps way
+
+Credit: @frank-bee
+
+
+```yaml
+provider "aws" {
+  region = "us-gov-west-1" # AWS GovCloud (US) region
+}
+
+data "aws_ami" "ubuntu" {
+  most_recent = true
+  owners      = ["099720109477"] # Canonical's AWS account ID
+
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*"]
+  }
+}
+
+resource "aws_instance" "example" {
```

Comment:
Use null resources instead

Reply:
idea was to launch a real instance ,( run vim, ) check if is running and report this back to terraform ( via output).
afterwards destroy the instance in case vim was running.
but I didn't test it, just wrote the code "from head to terminal" ;-)

Reply:
@aslafy-z ?